monkey/monkey_island/cc/ui/src/components/utils/ServerUtils.tsx

```diff
@@ -61,6 +61,16 @@ export function getMachineHostname(machine): string {
   return hostname;
 }
 
+export function getMachineFromID(id, machines) {
```

Comment:
should be `getMachineByID`, because we're not creating a machine FROM id, we're fetching it from the list BY it's id. Same goes for `getMachineFromIP`